lib/client/http.js

```diff
@@ -27,6 +27,7 @@ HttpClient = Client.extend(
             assert(_.isString(options.username), "options.username is expected to be a string");
             assert(_.isString(options.password), "options.password is expected to be a string");
             assert(_.isString(options.database), "options.database is expected to be a string");
+            assert(_.isNumber(options.max_batch), "options.max_batch is expected to be a number");
```

Comment:
I have a question:

If using `createClient()` without passing in `max_batch`, the value is defaulted to `5000` already because it comes from the `Client` class. The only way for this to be undefined is if someone instantiates `HttpClient` manually.

Should `HttpClient` require a value for `max_batch` if instantiated manually, or should it be optional there as well? `max_batch` is already used in the request batching, so a value is required, whether defaulted or not (currently).